quarkchain/evm/specials.py

```diff
@@ -322,6 +322,18 @@ def proc_mint_mnt(ext, msg):
     return 1, msg.gas - gascost, [0] * 31 + [1]
 
 
+def proc_balance_mnt(ext, msg):
+    address = msg.data.extract32(0)
+    token_id = msg.data.extract32(32)
+    # Should increase gas cost if having too many tokens
+    gascost = 400
```

Comment:
@qizhou what do you think? in the future we may have native tokens stored in trie